### PR TITLE
 After testing, you can use #575

### PR DESCRIPTION
**Pull request by cjmxp**
After testing, you can use #575

### CODE REVIEW

File: haxepunk/graphics/hardware/HardwareRenderer.hx [hunk 1]
```diff
@@ -70,9 +70,9 @@ class HardwareRenderer
 
 	public function new()
 	{
-		#if ios
-		defaultFramebuffer = new GLFramebuffer(GL.version, GL.getParameter(GL.FRAMEBUFFER_BINDING));
-		#end
+		//#if ios
+		//defaultFramebuffer = GL.createFramebuffer();
+		//#end
```
Comment (TheSpydog) [inline haxepunk/graphics/hardware/HardwareRenderer.hx]: Instead of commenting out, why not just delete these lines if they're not needed anymore?

**Comment by TheSpydog** [comment]
I haven't gotten a chance to verify the changes myself, but I'm curious how you tested this? This is a surprisingly simple change, which is great if everything works as it should. :)

**Comment by cjmxp** [comment]
I tested it on the simulator

**Comment by bendmorris** [comment]
I will be surprised if bindFrameBuffer on null suddenly works on iOS. Can anyone confirm this on device?

**Comment by TheSpydog** [comment]
This does not work on device.

**Comment by bendmorris** [comment]
Awesome, thanks to both of you. I'm going to clean up the implementation prior to merging.

**Comment by bendmorris** [comment]
>I am a derp and used an old version of Lime. This does actually work on device.

Which version did it work with, and which did it not work with?

**Comment by bendmorris** [comment]
I applied this fix. Thanks!

**Pull request closed without merging**